Samples/AudioCreation/cs/AudioCreation/BundleArtifacts/x64.txt

```diff
@@ -0,0 +1,2 @@
+MainPackage=C:\Git\Windows-universal-samples\Samples\AudioCreation\cs\AudioCreation\bin\x64\Debug\AudioCreation_0.1.0.0_x64_Debug.appx
```

Comment:
I suspect this is an auto-generated that is not supposed to be part of the commit. (If you think this file needs to be included, then what about the x86 and ARM versions of this file?)